docs/_posts/2016-09-27-leaflet-1.0-final.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: The wait is over. Meet Leaflet 1.0.
+description: Today marks the final release of Leaflet 1.0. It's a culmination of several years of work and the fastest, most stable and polished Leaflet release ever.
+author: Vladimir Agafonkin
+authorsite: http://agafonkin.com/en
+---
+
+Today marks the final release of Leaflet 1.0. It's a culmination of several years of work by dozens of contributors from all over the world. This is the fastest, most stable and polished Leaflet release ever.
+
+[The changelog](https://github.com/Leaflet/Leaflet/blob/master/CHANGELOG.md) contains _over 400 changes_ compared to the previous stable, v0.7.7. The highlights include:
+
+- Huge performance improvements in all aspects of the library and vector layers in particular.
+- New `L.Tooltip` class for all your dynamic labeling needs.
+- Flyover animations (zooming and panning in a curve with `map.flyTo`)
+- Fractional zoom level support (`map.setZoom(12.34`).
+- Much better tile loading algorithm with less flickering.
+- Custom pane management (including multiple vector layer panes and interleaving vectors and tile layers).
+- Better support for non-standard projections.
+- More accessibility features.
+- Vastly improved documentation.
+- _Tons_ of bugfixes and stability improvements.
+
+This release wouldn't be possible without such an amazing, diverse, loving community. We're incredibly proud to be a part of it. Thank you very much for your kind support!
+
+![](http://www.reactiongifs.com/r/msy.gif)
```

Comment:
We used this GIF already in a previous beta release blog post. Should be changed.